src/mmw/npm-shrinkwrap.json

```diff
@@ -44,6 +44,844 @@
       "from": "bootstrap-select@git://github.com/azavea/bootstrap-select",
       "resolved": "git://github.com/azavea/bootstrap-select#f1755efa102a41e43fc367ba36ce905b8f2b90e1"
     },
+    "bootstrap-table": {
```

Comment:
It really has none, but they didn't use `devDependencies` in the package.json: https://github.com/wenzhixin/bootstrap-table/blob/master/package.json#L9-L16